### PR TITLE
Navigator/tab bar

### PR DESCRIPTION
**Pull request by EstebanIndiveri**
## Summary
Added new TabNavigator, Library changed to LibraryStack, added news screens, created a custom TabBar,

### Android

<a href="https://ibb.co/sbZyHW7">Screenshot-1620341313</a>

<a href="https://ibb.co/JmrwfqR">Screenshot-1620341337</a>

#### Galaxy S (or similar)

<a href="https://ibb.co/8KP9zS3">Screenshot-1620341358</a><br /><a target='_blank' href='https://es.imgbb.com/'>picter com</a><br />

<a href="https://ibb.co/Dks57Rc">Screenshot-1620341366</a>

#### Pixel 2/3 (or similar)
<a href="https://ibb.co/0mqgTZF">Screenshot-1620341377</a>
<a href="https://ibb.co/N32Q9Wd">Screenshot-2021-05-06-19-51-46-324-com-wbooks</a>
### iOS
<a href="https://ibb.co/r706C4V">Simulator-Screen-Shot-i-Phone-SE-2nd-generation-2021-05-06-at-19-35-00</a>
<a href="https://ibb.co/WfHH3KC">Simulator-Screen-Shot-i-Phone-SE-2nd-generation-2021-05-06-at-19-35-07</a>

#### iPhone SE (or similar)

<a href="https://ibb.co/5xPzVDv">Simulator-Screen-Shot-i-Phone-8-2021-05-06-at-19-38-05</a>

<a href="https://ibb.co/hyhCpPN">Simulator-Screen-Shot-i-Phone-8-2021-05-06-at-19-38-10</a>

#### iPhone 6/7/8 (or similar)

<a href="https://ibb.co/B4Q8hWq">Simulator-Screen-Shot-i-Phone-12-2021-05-06-at-19-37-48</a>

<a href="https://ibb.co/8zWhS2C">Simulator-Screen-Shot-i-Phone-12-2021-05-06-at-19-37-57</a>

## Trello Card
[Trello TabBar card](https://trello.com/c/uktPxaGM/18-n-8-tab-bar)

### CODE REVIEW

File: wbooks/src/app/components/Tab/index.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import Routes from '@constants/routes';
+
+import styles from './styles';
+
+interface Props {
+  tab: {
+    key: string;
+    name: string;
+  };
+  onPress(): void;
```
Comment (andreszuluaga-wolox) [inline wbooks/src/app/components/Tab/index.tsx]: `onPress: () => void;` the functions shold be arrow funtions

File: wbooks/src/app/components/Tab/styles.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { StyleSheet } from 'react-native';
+import { textActiveTab, textInactiveTab } from '@constants/colors';
```
Comment (andreszuluaga-wolox) [inline wbooks/src/app/components/Tab/styles.js]: change by names more explicit

File: wbooks/src/app/components/TabBar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import Routes from '@constants/routes';
+import React, { ReactElement, useState } from 'react';
+import { View } from 'react-native';
+import Tab from '@components/Tab';
+
+import styles from './styles';
+
+interface Props {
+  navigation: any;
```
Comment (andreszuluaga-wolox) [inline wbooks/src/app/components/TabBar/index.tsx]: replace any

File: wbooks/src/app/components/TabBar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  state: {
+    routes: any;
```
Comment (andreszuluaga-wolox) [inline wbooks/src/app/components/TabBar/index.tsx]: same

File: wbooks/src/app/components/TabBar/styles.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import { white } from '@constants/colors';
+import { StyleSheet, Dimensions, Platform } from 'react-native';
```
Comment (andreszuluaga-wolox) [inline wbooks/src/app/components/TabBar/styles.js]: this should be up:
first declare libs

File: wbooks/src/constants/colors.ts [hunk 6]
```diff
@@ -3,3 +3,5 @@ export const white = '#FFF';
 export const black = '#323232';
 export const gray = '#323232';
 export const whiteBlue = '#e5edf7';
+export const textActiveTab = '#1ec9e5';
```
Comment (andreszuluaga-wolox) [inline wbooks/src/constants/colors.ts]: use uppercase for Hex

File: wbooks/src/app/components/TabBar/styles.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import { StyleSheet, Dimensions } from 'react-native';
+import { cyan, lightGray } from '@constants/colors';
+
+const { width } = Dimensions.get('window');
+const styles = StyleSheet.create({
+  container: {
+    width,
```
Comment (mattgle) [inline wbooks/src/app/components/TabBar/styles.js]: Can't you just use `width: '100%'` ?

File: wbooks/src/app/navigator/index.tsx [hunk 8]
```diff
@@ -1,20 +1,32 @@
...
 import Routes from '@constants/routes';
-import { DETAIL_BOOK_OPTIONS, HOME_OPTIONS, STACK_NAVIGATOR_OPTIONS } from '@constants/screenOptions';
+import WishList from '@screens/WishList';
+import Suggest from '@screens/Suggest';
+import Rentals from '@screens/Rentals';
+import TabBar from '@components/TabBar';
+import { TAB_BAR_OPTIONS } from '@constants/screenOptions';
 
-const Stack = createStackNavigator();
+import LibraryStack from './LibraryStack';
```
Comment (mattgle) [inline wbooks/src/app/navigator/index.tsx]: Move libraryStack here. dont extract it

**Review by mattgle — CHANGES_REQUESTED** [review]

**Review by mattgle — APPROVED** [review]

**Pull request merged**